data/panel.css

```diff
@@ -59,15 +59,29 @@ video#video {
     background-repeat: no-repeat;
     background-size: cover;
     background: url('img/static.gif');
+    height: 100%;
+}
+
+.error-message-container {
+    background: #00000080;
```

Comment:
I'm not a CSS Wizard, but not sure on the `#RRGGBBAA` support yet.
After a bit of searching, MDN comes back with https://developer.mozilla.org/en-US/docs/Web/CSS/color_value which lists FF49+ for `#RRGGBBAA` versus FF3+ for `rgba()` syntax.

/cc My CSS Spirit Guide, Mr. @johngruen